src/js/twitter.js

```diff
@@ -1,3 +1,64 @@
+import { getNodesDeep } from './utils'
+
+const twitter_filter = (prev_tweet_count = 0) => {
+  const twitter_handles = []
+  const orgs = document.getElementsByClassName('org-info')
+
+  for (let i = 0; i < orgs.length; i++) {
+    const org_info = orgs[i].children
+    const children = [].slice.call(org_info)
+    const twitter_url = children.filter(
+      x => x.nodeName === 'A' && x.host === 'twitter.com'
+    )
+
+    if (twitter_url.length !== 0) {
+      const twitter_account = twitter_url[0].pathname.substr(1)
+      twitter_handles.push(twitter_account)
+    }
+  }
+
+  const iframe_id = document.getElementById('twitter-widget-0')
+
+  if (iframe_id !== null) {
+    if (typeof iframe_id.contentWindow !== 'undefined') {
+      const content_window_document = iframe_id.contentWindow
+      const tweet_authors = content_window_document.getElementsByClassName(
+        'TweetAuthor-screenName Identity-screenName'
+      )
+      const tweet_count = tweet_authors.length
+
+      const load_button = content_window_document.getElementsByClassName(
+        'timeline-LoadMore-prompt timeline-LoadMore-prompt--edge\
+         timeline-ShowMoreButton--edge timeline-ShowMoreButton customisable'
+      )[0]
+
+      if (typeof load_button !== 'undefined') {
```

Comment:
I personally prefer `if (load_button)` because it's shorter and cleaner. Not sure others' opinions on this though?

Reply:
No, that is not the problem. A `getElementByClassName` will always return a `HTMLCollection`. It is obvious neither `undefined` nor `null`. That `if` will be a tautology since `HTMLCollection` is `true`. Use this instead.

```javascript
if (load_button.length > 0) {
  // ...
}
```

Reply:
I'm forgot that he directly call getter `[0]` so my comment above not relevant now.